### ISSUE TITLE
Where are segnet maxpool indices leveraged ?

### ISSUE THREAD

**Issue by gnthibault** [divamgupta/image-segmentation-keras#249]
Hi all,
Very interesting  blog article and codebase, thank you.
I was interested in trying to compare the various segmentation strategies mentionned in the blog article, and realized that the overall effort needed to provide an exhaustive (or almost exhaustive) combination of encoder architecture + segmentation principle + pretrained weight was huge. So congratulations for the project.

I was especially interested in the following combination: resnet50 inside of segnet.
Then I started to study those two files:
https://github.com/divamgupta/image-segmentation-keras/blob/master/keras_segmentation/models/segnet.py
and
https://github.com/divamgupta/image-segmentation-keras/blob/master/keras_segmentation/models/resnet50.py

Looks like the segnet code is just using a bunch of UpSampling2D + ZeroPadding2D + Conv2D + BatchNormalization so almost exactly like the Unet bu without skip connection. If I am not mistaking, the maxpooling indices are missing ?

**Comment by gnthibault** [comment]
Ok, of course this has been already noticed in #236 and #17
